ramls/schema/effective-call-number-components.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Elements of a full call number generated from the item or holding",
+  "properties": {
+    "callNumber": {
+      "type": "string",
+      "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item."
+    },
+    "callNumberPrefix": {
```

Comment:
The `callNumber` prefix could be redundant here. Is it intended to differentiate between prefix / suffix for other elements?

Reply:
Hi @marcjohnson-kint, sorry did not get your question. My understanding that  `callNumberComponents`  object should have all the components that we derive in inventory (but we are committed to suffix/prefix an call number itself for now, since they are mostly completed in inventory).

Reply:
My comment is suggesting that there is no need to call the `suffix` the `callNumberSuffix` because it is in the `callNumberComponents` object.